### DISCUSSION TITLE
azd config set alpha.resourceGroupDeployments on

### DISCUSSION THREAD

**Discussion by BartNetJS** [Azure/azure-dev#3415] [Q&A] [answered]
Is there any time frame when resource group deployments will be final?

**Comment by johnnyreilly** [comment]
I'm assuming @BartNetJS means when `resourceGroupDeployments` will no longer be alpha?  I'd love to know too!

**Reply by rajeshkamal5050** [reply]
Starting [September 1.10.X release](https://devblogs.microsoft.com/azure-sdk/azure-developer-cli-azd-september-2024/), resource group scoped deployments is now in Beta. NOTE: It was promoted to Beta with the introduction of deployment stacks support which addresses issues related to `azd down` during rg scoped deployments.

cc @ellismg @wbreza

**Reply by wbreza** [reply, marked as answer]
As of 1.10 resource groups is out of alpha and has been promoted to a beta feature.

We don’t have a specific timeframe for when the resource group deployments will be finalized. It will move forward once we’ve gathered enough feedback to ensure everything is running smoothly. We’ll keep you informed as we progress.
